platforms/aws/giantnetes/variables.tf

```diff
@@ -42,14 +42,25 @@ variable "vault_instance_type" {
   default = "t2.medium"
 }
 
-variable "master_instance_type" {
-  type    = "string"
-  default = "m5.large"
+variable "master_instance" {
+  type = "map"
+
+  default = {
+    type                     = "m5.large"
+    mount_device_name_docker = "/dev/nvme1n1"
+    mount_device_name_etcd   = "/dev/nvme2n1"
+    volume_device_name_etcd  = "/dev/xvdh"
```

Comment:
Naming is confusing.  Why use `mount_` and `volume_` prefixes? `device_name_xxx` pretty enough. And i think you have to remove one for `volume_device_name_etcd`.

Reply:
I'll adjust naming. Also `volume_device_name_etcd` - that is aws volume attachment. Naming was mostly based on the PR https://github.com/giantswarm/giantnetes-terraform/pull/76

Changes in ignition files are also there. e.g. https://github.com/giantswarm/giantnetes-terraform/pull/90/files#diff-b3bb19fde7c1bd581f41bd9d02e41b7bR2307

Reply:
I see. Then it's fine.

Reply:
I did not know get it initially